object_analytics_node/src/movement/moving_object_node.cpp

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2018 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <class_loader/register_macro.hpp>
+#include <rclcpp/rclcpp.hpp>
+
+#include <cstdio>
+#include <memory>
+#include <string>
+#include "object_analytics_node/movement/moving_object_node.hpp"
+#include "object_analytics_msgs/msg/moving_object.hpp"
+#include "object_analytics_node/movement/param.hpp"
+
+using std::placeholders::_1;
+
+namespace object_analytics_node
+{
+namespace movement
+{
+MovementNode::MovementNode()
+: Node("MovementNode")
+{
+  // Force flush of the stdout buffer.
+  moving_objects_pub_ =
+    create_publisher<MovingObjectMsg>("/object_analytics/movement", 10);
```

Comment:
I would suggest define a variable for the topic "/object_analytics/movement" in const.cpp or remap.cpp, and use variable instead.